docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+# First Steps with DataFrames
+
+The first and imporatant part to work with DataFrames.jl, is to install the `DataFrames.jl`
+package which is available thorugh the Julia package system and can be installed using the
+following commands:
+
+```julia
+julia> using Pkg
+julia> Pkg.add("DataFrames")
+```
+
+Or,
+
+```julia
+julia> ]
+
+julia> add DataFrames
+```
+
+To make sure everything works as expected, try to load the package and if you have the time execute
+its test suits:
+```julia
+julia> using DataFrames
+julia> using Pkg
+julia> Pkg.test("DataFrames")
+```
+
+Throughout the rest of the tutorial we will assume that you have installed the DataFrames package and
+have already typed `using DataFrames` to bring all of the relevant variables into your current namespace.
+Let's get started by loading the DataFrames package:
+```julia
+julia> using DataFrames
+```
+
+The object of the `DataFrame` type represent a data table as a series of vectors, each correspondng to a
+column or variable.
+
+# Constructors and basic utility functions
+
+## Constructors
+
+In this section you will see many ways to create a DataFrame using `DataFrame()` constructor.
+First, we could create an empty DataFrame:
+```jldoctest
+julia> using DataFrames
+
+julia> df = DataFrame()
+0×0 DataFrame
+```
+
+Or, we could call the constructor using keyword arguments to add columns to the DataFrame:
+```jldoctest
+julia> using DataFrames
+
+julia> df = DataFrame(A=1:3, B=5:7, fixed=1)
+3×3 DataFrame
+ Row │ A      B      fixed
+     │ Int64  Int64  Int64
+─────┼─────────────────────
+   1 │     1      5      1
+   2 │     2      6      1
+   3 │     3      7      1
+```
+note in column `:fixed` that scalars get automatically broadcasted.
+
+Now, we will explore how to load a CSV file into into a DataFrame. Unlike Python's Pandas `read_csv`
+the functions in Julia are separated into two modules - `CSV.jl` and `DataFrames.jl`. As the first
+step, you have to declare the libraries you will use. In our case `CSV` and `DataFrames`. In order to
+turn the `CSV.File` to a DataFrame you have to pass it to the `DataFrames.DataFrame` object. You can
+wrap DataFrame around the `CSV.read(path; kwargs)`.
+
+```jldoctest dataframe
+julia> using DataFrames
+
+julia> using CSV
```

Comment:
probably the user should be instructed how to install it and what version is expected

Reply:
It has been mentioned in the starting.

Reply:
this is a tutorial so better make it step-by-step